libpromises/override_fsattrs.h

```diff
@@ -33,26 +33,32 @@
  * @param orig The original file (may be immutable)
  * @param copy Updated to contain the filename of the mutable copy
  * @param copy_len The size of the buffer to store the filename of the copy
+ * @param override Whether to actually do override (original filename is
```

Comment:
do you mean overwrite? as in directly manipulate the file instead of making the temp copy?

Reply:
I see what override means now... override the immutable bit while you do the operation.